live-hls.js

```diff
@@ -109,6 +113,12 @@ getHeaderObjects = function(fileContent) {
       continue;
     }
 
+    if (lines[i].indexOf('#EXT-X-PROGRAM-DATE-TIME') === 0) {
```

Comment:
`EXT-X-PROGRAM-DATE-TIME` is a segment header tags, so it should be parsed in `getSegmentHeader`